recipes/conda/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.0.6" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  fn: conda-{{ version }}.tar.gz
+  url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
+  sha256: 4ca7ec875de13519aeb3eebc9beca23adf3c41e8852c186deb503e8bc2282c40
+
+build:
```

Comment:
activate and deactivate both need to be in always_include (win needs them for bash support).  Windows also needs the .bat files.  There's a fish script, but I don't know where it goes.  Issue here: https://github.com/conda/conda/issues/2611